cmd/fireeye2nvd/fireeye2nvd.go

```diff
@@ -55,18 +56,21 @@ func main() {
 	sinceUnix := flag.Int64("since_unix", 0, "Unix timestamp since when should we download. If not set, downloads all available data")
 	sinceDuration := flag.String("since", "", "Golang duration string, overrides -since_unix flag")
 	dontConvert := flag.Bool("dont_convert", false, "Should the feed be converted to NVD format or not")
+	stats.Global.AddFlags()
```

Comment:
This additional level of indirection is kind of ugly, why not just `stats.AddFlags()`, `stats.WriteAndLogError()` etc.?